tests/v2/validation/rbac/rbac_test.go

```diff
@@ -390,12 +406,35 @@ func (rb *RBTestSuite) ValidateCannotAddMPMsAsProjectOwner() {
 	assert.Equal(rb.T(), "422 Unprocessable Entity", errorMsg[1])
 
 	addNewUserAsPOClient, err = addNewUserAsPOClient.ReLogin()
+	require.NoError(rb.T(), err)
 
 	userGetProject, err := projects.GetProjectList(addNewUserAsPOClient, rb.cluster.ID)
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 0, len(userGetProject.Data))
 }
 
+func (rb *RBTestSuite) ValidateListGlobalSettings() {
+	adminListSettings, err := listGlobalSettings(rb.steveAdminClient)
+	require.NoError(rb.T(), err)
+	resAdminListSettings, err := listGlobalSettings(rb.steveStdUserclient)
+	require.NoError(rb.T(), err)
+
+	assert.Equal(rb.T(), len(adminListSettings), len(resAdminListSettings))
+	assert.Equal(rb.T(), adminListSettings, resAdminListSettings)
+}
+
+func (rb *RBTestSuite) ValidateEditGlobalSettings() {
+	globalSettingByID, err := rb.steveStdUserclient.SteveType("management.cattle.io.setting").ByID(kubeConfigTokenSetting)
+	require.NoError(rb.T(), err)
+
+	updatedSettingValue, err := editGlobalSettings(rb.steveStdUserclient, globalSettingByID, "3")
```

Comment:
I can't tell which setting is this changing to `3`.

`ValidateEditGlobalSetting` is vague to show which setting is getting updated.

Reply:
Agreed I think maybe we can call the returned setting `kubeConfigTokenSetting` and its ID as `kubeConfigTokenSettingID`. Just to show an example please feel free to name however you see fit.